### PR TITLE
Not ready for review: Adding assertions for default and t3channel node port verification

### PR DESCRIPTION
**Pull request by vanajamukkara**
Adding assertions for default and t3channel node port verification for adminservice

Jenkins run is clean on both cluster-10 and on kind cluster,

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/84/

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/94/

Please wait to review this PR. See my comment on Sankar's PR - https://github.com/oracle/weblogic-kubernetes-operator/pull/1648#discussion_r425770296. If that is done, this PR is not needed.

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java [hunk 1]
```diff
@@ -321,7 +321,8 @@ public void testCreateMiiConfigMapDomain() {
       checkServiceCreated(managedServerPrefix + i, domainNamespace);
     }
 
-    int adminServiceNodePort = getAdminServiceNodePort(adminServerPodName + "-external", null, domainNamespace);
+    int adminServiceNodePort = getAdminServiceNodePort(adminServerPodName + "-external",
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java:324]: I changed this method to have generic name. Its in PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1648

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java [hunk 2]
```diff
@@ -85,19 +88,57 @@ public static boolean doesCrdExist() throws ApiException {
...
+   * the node port.
+   *
+   * @param nodePort admin default/t3channel node port
+   * @param username admin user name to access the REST url
+   * @param password admin password to access the REST url
+   * @return true if admin REST url is accessible using node port
+   */
+  public static boolean adminNodePortAccessible(int nodePort, String username, String password) {
+    StringBuffer cmd = new StringBuffer();
+    cmd.append("curl --silent --show-error --noproxy ")
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:102]: we should move away from curl and start using java apis
https://openjdk.java.net/groups/net/httpclient/intro.html

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java (hunk 1 above)
Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java:324]: Ok, will resolve when its merged.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java (hunk 2 above)
Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:102]: sure, will make the changes in separate PR.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java [hunk 3]
```diff
@@ -64,9 +67,9 @@ public static boolean doesCrdExist() throws ApiException {
   /**
    * Checks if weblogic.oracle CRD domain object exists.
    *
-   * @param domainUid domain UID of the domain object
+   * @param domainUid     domain UID of the domain object
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:70]: Remove extra blank space between words in a line

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java [hunk 4]
```diff
@@ -64,9 +67,9 @@ public static boolean doesCrdExist() throws ApiException {
...
    * @param domainVersion version value for Kind Domain
-   * @param namespace in which the domain object exists
+   * @param namespace     in which the domain object exists
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:72]: The sentence should be read as follows ...
@param namespace  namespace  in which the domain object exists

The first word is the exact name of the parameter and the rest of the sentence is to describe the parameter

**Comment by vanajamukkara** [comment]
@anpanigr thanks for the review comments, but this PR is not ready for review, please see the PR description.

**Comment by vanajamukkara** [comment]
Closing this PR as Sankar moved the console check into assertion to verify node port in PR#1648

**Pull request closed without merging**